tests/cpptests/test_cpp_query_validation.cpp

```diff
@@ -51,34 +79,31 @@ TEST_F(QueryValidationTest, testInvalidVectorFilter) {
   RedisSearchCtx ctx = SEARCH_CTX_STATIC(NULL, (IndexSpec *)StrongRef_Get(ref));
 
   // Invalid queries with KNN
-  assertInvalidVectorFilter("*=>[KNN 10 @vec_field $BLOB]", ctx);
-  assertInvalidVectorFilter("@title:hello =>[KNN 10 @vec_field $BLOB]", ctx);
+  assertInvalidHybridVectorFilterQuery("*=>[KNN 10 @v $BLOB]", ctx);
+  assertInvalidHybridVectorFilterQuery("@title:hello =>[KNN 10 @v2 $BLOB]", ctx);
```

Comment:
I see `v` and `v2` are the same - any reason for testing/creating both?

Reply:
No. Test refactored to use only `v`. Thanks.